Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean

```diff
@@ -542,6 +542,26 @@ variable {R : Type u} [CommRing R] (E : EllipticCurve R)
 def j : R :=
   E.Δ'⁻¹ * E.c₄ ^ 3
 
+theorem j_eq_zero_iff' : E.j = 0 ↔ E.c₄ ^ 3 = 0 := by
+  simp only [EllipticCurve.j, Units.mul_right_eq_zero]
+
+alias ⟨_, j_eq_zero'⟩ := j_eq_zero_iff'
```

Comment:
I'm not convinced we need the directional aliases, since we can just do `E.j_eq_zero_iff'.mpr` etc.

Reply:
Fine.